Dockerfile

```diff
@@ -9,10 +11,11 @@ RUN apt-get update && apt-get install -y \
   zlib1g-dev
   
 RUN pip install --upgrade pip
-COPY requirements.txt ./
-COPY setup ./
+
+COPY requirements.txt config.ini.example setup rosie.py ./
```

Comment:
I tend to avoid multiple tasks in a single line/command – if something is broken (eg missing file) it's way easier to find out what's wrong.